### DISCUSSION TITLE
Bump axios to "^1.0.0"

### DISCUSSION THREAD

**Discussion by Mosaab-Emam** [quasarframework/quasar#15161] [Ideas / Proposals] [answered]
[Axios](https://github.com/axios/axios) hit version `1.0` in October 2022, however, when choosing axios during creating an app with the cli, the version of axios specified in package.json is still "^0.21.1".

I suggest bumping the version used to "^1.0.0"  so the latest version of axios will be installed.

**Comment by MilosPaunovic** [comment]
PRs are always welcome! 🙂

**Reply by Mosaab-Emam** [reply]
Will do.

**Reply by Mosaab-Emam** [reply]
@MilosPaunovic #15165 

Done! Please review. :)

**Comment by Mosaab-Emam** [comment, marked as answer]
PR opened #15165.

@rstoenescu
